File: c++/Assignment1/Assignment1/largest.cpp
#include<iostream>
using namespace std;

int main()
{
	int a[] = {1662,33,211,599};
	int max = a[0];
	for (int i = 0; i < sizeof(a) / sizeof(a[0]); i++)
	{

		if (a[i] > max)
		{
			max = a[i];
		}
		else
		{
			max = max;
		}
	}
	cout << "Largest number in an array is : " << max;
	return 0;

}

File: java/Largest.java
import java.util.*;

class Largest {
	
	public static void main(String[] args) {
		
		Scanner s = new Scanner(System.in);

		System.out.println("Enter 1st no- ");
		int a = s.nextInt();
		System.out.println("Enter 2nd no- ");
		int b = s.nextInt();
		System.out.println("Enter 3rd no- ");
		int c = s.nextInt();

		if (a>=b && a >= c)
		{
			System.out.println("Largest number is - " + a);
		}
		else if (b>=c && b>=a)
		{
			System.out.println("Largest number is - " + b);
		}
		else{
			System.out.println("Largest number is - " + c);
		}

	}
}